CHANGELOG.md

```diff
@@ -60,9 +60,9 @@ _None_
 
 ### Internal Changes
 
-* Integrated Danger to help on contributions by giving automatic feedback.  
+* Integrated Danger to help on contributions by giving automatic feedback.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/3e528d2352073cbd935a3bba49bd233416593357/CHANGELOG.md#L63"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>